### ISSUE TITLE
No trigger for <c-r> in command mode

### ISSUE THREAD

**Issue by Tanath** [junegunn/vim-peekaboo#54]
Peekaboo doesn't trigger on <kbd>ctrl</kbd>+<kbd>r</kbd> in `:command` mode.

**Comment by ajkavanagh** [comment]
Yes, doesn't work for me either.  Not sure if it's supposed to?

**Comment by TheJoeSchr** [comment]
yeah, it was the first I checked as well, would be great if this could work

**Comment by TheJoeSchr** [comment]
I could "fix" it by using `which-key` (https://github.com/folke/which-key.nvim)
